sdk/python/kfp/dsl/_container_op.py

```diff
@@ -1023,7 +1023,7 @@ def _decorated(*args, **kwargs):
 
         # attributes specific to `ContainerOp`
         self.file_outputs = file_outputs
-        self.output_artifact_paths = output_artifact_paths or {}
+        self._output_artifact_paths = output_artifact_paths or {}
```

Comment:
Are there any functions that update the output_artifact_paths? If not, maybe we should as we make it private. WDYT?

Reply:
> Are there any functions that update the output_artifact_paths? If not, maybe we should as we make it private. WDYT?

I think this PR does exactly that. Did you mean `file_outputs`?